### PR TITLE
IOS collection prep for v1.3.0 tagging

### PR DESCRIPTION
**Pull request by justjais**
##### SUMMARY

IOS collection prep for v1.3.0 tagging

##### ISSUE TYPE

- Docs Pull Request

##### COMPONENT NAME

ios

##### ADDITIONAL INFORMATION

```paste below

```

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -8,7 +8,7 @@ This collection has been tested against Cisco IOSv version 15.2 on VIRL.
 <!--start requires_ansible-->
 ## Ansible version compatibility
 
-This collection has been tested against following Ansible versions: **>=2.9.10,<2.11**.
+This collection has been tested against following Ansible versions: **>=2.9.10**.
```
Comment (NilashishC) [inline README.md:11]: This should not have changed, should it?

**Review by NilashishC — APPROVED** [review]

**Comment by justjais** [comment]
recheck

File: meta/runtime.yml [hunk 2]
```diff
@@ -1,5 +1,5 @@
 ---
-requires_ansible: '>=2.9.10'
+requires_ansible: '>=2.9.10,<2.11'
```
Comment (NilashishC) [inline meta/runtime.yml:2]: This is currently failing the EE test.

**Comment by justjais** [comment]
recheck

**Pull request merged**